src/components/CollectionRow/CollectionRow.js

```diff
@@ -100,6 +105,18 @@ const CollectionRow = ({
           <Text style={styles.labelText}>{label}</Text>
         </View>
         <View style={styles.iconContainer}>
+
+          {collection.current &&
```

Comment:
add optional chaining (`?.`) to prevent breakage while in use. So this should be `collection?.current`. This should be applied whenever a value could be `undefined`.

Reply:
resolved